releng/org.eclipse.equinox.releng/EquinoxConfiguration.setup

```diff
@@ -61,29 +61,24 @@
             value="Equinox"/>
       </setupTask>
     </setupTask>
-    <setupTask
-        xsi:type="setup:VariableTask"
-        name="eclipse.git.authentication.style"
-        defaultValue="anonymous"/>
```

Comment:
@merks is it necessary to define this variable? As far as I can tell, the default is already `anonymous` anyways:

Reply:
I think not. I think I changed this from the olden days when one typically wanted a gerrit URI for contribution, but now one can use anonymous and commit to a fork, and even the committers do that.

Reply:
Yes, personally I usually fetch from the upstream/original repo using https and push to my fork via SSH.

Ok, good. Then we can probably remove it from all other configurations. I can take care of that.
Thanks.